apps/foundation-demo/src/app/core/footer/footer.component.scss

```diff
@@ -2,18 +2,20 @@
 
 .footer {
   display: grid;
-  grid-template-columns: repeat(2, 1fr) 0.6fr;
-  grid-template-rows: auto;
-  grid-row-gap: 20px;
-  margin: 35px 15px;
+  grid-template-columns: 1fr;
+  grid-gap: 25px;
+  padding: 50px 30px;
 
   @include breakpoint(medium) {
-    margin: 0 50px 100px 50px;
-    grid-template-columns: repeat(2, 1fr) repeat(2, 0.75fr);
+    grid-template-columns: 1fr 1fr;
+    padding: 75px 50px;
   }
 
   @include breakpoint(large) {
-    margin: 0 150px 100px 200px;
     grid-template-columns: repeat(4, 1fr);
   }
+
+  @include breakpoint(xlarge) {
+    padding: 75px 0;
```

Comment:
Are you sure you want no right or left padding? It looks off to me at anything wider than 1200px.

Reply:
same with the newsletter styles

Reply:
Yes, I don't want left/right padding. It'll just contain the content more than the specified `daff-container` size. I only added padding to the smaller screen sizes so the content won't be edge to edge.

Reply:
But because footer and newsletter are in `template.html`, doesn't that mean that they'll never be affected by `daff-container`? Because `daff-container` will wrap the `<ng-router></ng-router>` in:
```
<div class="template">
    <sidebar-viewport-container>
        <header-container></header-container>
        <router-outlet></router-outlet>
        <daff-newsletter></daff-newsletter>
        <footer></footer>
    </sidebar-viewport-container>
</div>
```
? Unless `template` will be wrapped in a `daff-container` instead of it's children.

Reply:
If you check the `newsletter.component.html` and `footer.component.html` file, you'll see that I wrapped the content in both with `daff-container`.

Reply:
If you check the `newsletter.component.html` and `footer.component.html` file, you'll see that I wrapped the content in both with `daff-container`.

Reply:
oh gotcha. I just take issue with the `daff-container` then. When the width is 1200px, there's no margin or padding left or right. But if that's ok with you, then it's ok with me